grafana/config.libsonnet

```diff
@@ -0,0 +1,56 @@
+local k = import 'k.libsonnet';
+{
+  _config+:: {
+    grafana+: {
+      rootUrl: error 'Root URL required',
+      provisioningDir: '/etc/grafana/provisioning',
+      dashboardConfigMapCount: 1,
+      labels+: {
+        dashboards: [],
+        datasources: [],
+        notificationChannels: [],
+      },
+      grafana_ini+: {
+        sections+: {
+          server: {
+            http_port: 3000,
+            router_logging: true,
+            root_url: $._config.grafana.rootUrl,
+          },
+          analytics: {
+            reporting_enabled: false,
+          },
+          users: {
+            default_theme: 'light',
+          },
+          explore+: {
+            enabled: true,
+          },
+        },
+      },
+    },
+  },
+
+  withGrafanaIniConfig(config):: {
```

Comment:
Does this imply that `config` arg is in an ini-format?

Reply:
The things I'm trying to name are:
* generic config for Grafana itself, e.g. port number
* configurations that go into the `/etc/grafana-config/grafana.ini` file. The above function, and the `grafana_ini` element are intended for this destination. Open to other suggestions for better names.